app/models/user.rb

```diff
@@ -316,9 +316,15 @@ def liked_pages
   end
 
   def send_digest_email
-    top_picks = content_followed_in_period(1.week.ago, Time.now)
-    if top_picks.count.positive?
-      SubscriptionMailer.send_digest(id, top_picks).deliver_now
+    if self.has_tag('digest:daily')
+      top_picks = content_followed_in_period(Time.now - 1.day, Time.now)
+      k = 0
+    elsif self.has_tag('digest:weekly')
```

Comment:
Aha - so if a user has /neither/ tag set, we are still trying to call top_picks. We should set it to `[]` by default? We could define it above as `top_picks = []` before these IFs?

Reply:
Yes I thought of doing the same, but I didn't. So sorry!

Reply:
no worries!